includes/definitions.inc.php

```diff
@@ -2228,6 +2228,11 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+$os = 'trapeze';
+$config['os'][$os]['text']             = 'Juniper Wireless';
+$config['os'][$os]['type']             = 'wireless';
+$config['os'][$os]['icon']             = 'junos';
+
```

Comment:
Other options can be added / changed later but it's best to have at least some set by default such as traffic, mem and cpu